packages/nc-gui/components/virtual-cell/components/LinkRecordDropdown.vue

```diff
@@ -88,9 +88,9 @@ watch([ncLinksDropdownRef, isOpen], () => {
   @apply h-[412px] w-[540px] xs:(w-[90vw] min-h-[312px] h-[312px]);
   overflow-y: auto;
   overflow-x: hidden;
-  resize: vertical;
+  resize: both;
   min-height: 412px;
-  max-height: 700px;
-  max-width: 540px;
+  max-height: 720px;
```

Comment:
@rameshmane7218 , yes, I just checked that it's lagging, both the x and the y axis, but ig there's an issue in the max-width only(not really sure of this)? Because, I just checked by scrolling the box to the left and this showed up <br>
![image](https://github.com/user-attachments/assets/88c8753b-6741-47bf-852e-d24f0b1bc8ab)

Actually, for the lagging part, I'm not able to figure out why this issue is there? Could you please guide regarding this